server/client.go

```diff
@@ -585,8 +591,8 @@ func (c *client) initClient() {
 	c.cid = atomic.AddUint64(&s.gcid, 1)
 
 	// Outbound data structure setup
-	c.out.sz = startBufSize
```

Comment:
Having very small buffers helps with idle connections, but the pool may do just as well. But this could cause a run on the bank so to speak in certain situations, like sending a ping to a large number of clients who send a response. They only do this once in a while, but each response would load the buffer. It is 512 now but that is also limiting performance quite a bit.